3001-4000/LIT3167Eleutheris.xml

```diff
@@ -39,31 +55,72 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </xi:include>
       </encodingDesc>
       <profileDesc>
+         <abstract><p>(Synaxary version) Eleutherus was brought up by a God-fearing bishop
```

Comment:
This reads like it was copied from where, it would maybe be good to indicate the source